utils/eventtracking.js

```diff
@@ -82,6 +86,16 @@ const track = async (eventType, eventProperties, options) => {
     }
 };
 
+async function getPublicIp() {
+    return fetch(
+        "https://checkip.amazonaws.com/"
+    ).then(
+        res => res.text()
+    ).then(
+        ip => ip.trim()
+    );
+}
+
 const getEventTrackingConsent = async () => {
     return askYesNoQuestion(
         chalk`Please help us to collect data on near-cli usage to improve developer experience. ` +
```

Comment:
Is this the only question for getting consent? Do we have separate one when we grab their account id?

Reply:
This is the only question that is getting asked on the first run.
We are not collecting NEAR Account ID, we are collecting generated user ID. It's unique for each machine and can be easily deleted by the user.